react-useEffect/react-effects-and-fetch_api-status/src/components/StatusCheck/index.js

```diff
@@ -1,26 +1,23 @@
+import { useEffect, useState } from "react";
 import "./StatusCheck.css";
 
 const apiStatusUrl = "https://example-apis.vercel.app/api/status";
 
 export default function StatusCheck() {
-  const statusIcon = "⁉️";
-  // Something needs to change here…
-  // ↙️
-  function handleCheckApiStatus() {
-    /**
-     * Hint 1:
-     * Use the `fetch()` function and pass the `apiStatusUrl` into it
-     *
-     * Hint 2:
-     * The fetch function returns a promise which resolves to a Response
-     * object once it is ready.
-     *
-     * Hint 3:
-     * The Response object has a `ok` property which is true if the response
-     * is okay and false if it is not.
-     **/
-    // --v-- write your code here --v--
-    // --^-- write your code here --^--
+  const [statusIcon, setStatusIcon] = useState("⁉️");
+  const [data, setData] = useState("");
+
+  async function handleCheckApiStatus(event) {
+    try {
+      event.target.textContent = "⏳";
```

Comment:
This changes the button text content from "Check API Status" to   "⏳". As I understood from the requirements a statusIcon  "⏳" should be displayed instead of "⁉️", "✅" , "❌"